locales/en-US/app.ftl

```diff
@@ -687,6 +687,14 @@ NumberFormat--short-date = { SHORTDATE($date) }
 PanelSearch--search-field-hint =
     Did you know you can use the comma (,) to search using several terms?
 
+## Profile Name Button
```

Comment:
hey @flodolo , do you think this needs some more comments, for example to make explicit that only title and aria-label should be translated, but the string itself should stay empty?

Reply:
ah perfect, thanks!